File: 프로그래머스/0/181899. 카운트 다운/카운트 다운.js
function solution(start_num, end_num) {
    let answer = new Array(start_num-end_num+1);
    let n =0;
    for(let i=start_num; i>= end_num; i--) {
        answer[n] = i;
        n++;
    }
    return answer;
}

File: 프로그래머스/0/181899. 카운트 다운/카운트 다운.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int start_num, int end_num) {
    vector<int> answer(start_num-end_num+1);
    for(int i=0; i <= start_num-end_num ; i++) {
            answer[i] = start_num-i;
    }
    return answer;
}